### PR TITLE
[#892] open cases view

### PR DESCRIPTION
**Pull request by annashamray**

### CODE REVIEW

**Comment by annashamray** [comment]
I separated case list tests into 3 files and refactored them a little bit in order to test 2 views instead of 1
Sorry about that

**Review by alextreme — APPROVED** [review]

File: src/open_inwoner/accounts/views/cases.py [hunk 1]
```diff
@@ -145,27 +131,67 @@ def get_cases(self):
...
+    def get_cases(self):
+        all_cases = super().get_cases()
+
+        cases = [case for case in all_cases if case.einddatum]
+        cases.sort(key=lambda case: case.einddatum, reverse=True)
+        return cases
+
+    def get_anchors(self) -> list:
+        return [
+            (reverse("accounts:my_closed_cases"), _("Lopende aanvragen")),
```
Comment (vaszig) [inline src/open_inwoner/accounts/views/cases.py:189]: This should redirect to `"accounts:my_open_cases"` , right?Now both anchors redirect to closed cases.

**Review by vaszig — COMMENTED** [review]
Except for a small change in anchors it looks nice. Tests refactoring also looks nice.

File: src/open_inwoner/accounts/views/cases.py (hunk 1 above)
Comment (annashamray) [reply src/open_inwoner/accounts/views/cases.py:189]: Nice catch!

**Review by Bartvaderkin — APPROVED** [review]
:+1:

**Pull request merged**